implementations/rust/ockam/ockam_abac/src/policy.rs

```diff
@@ -16,6 +17,7 @@ use tracing as log;
 /// Attributes come from a pre-populated environment and are augmented
 /// by subject attributes from credential data.
 pub struct PolicyAccessControl {
+    node_name: String,
```

Comment:
Let's use node_name from the SqlxDatabase

Reply:
Done